backend/src/routes/api/nb-events/index.ts

```diff
@@ -10,6 +10,10 @@ export default async (fastify: FastifyInstance): Promise<void> => {
           namespace: string;
           notebookName: string;
         };
+        Querystring: {
```

Comment:
I had several 20kb requests because I cause a lot of recent events -- in the current structure I'd imagine we can definitely get into a situation where it's not ideal. I do not think it is critical -- which is why I didn't stick to fixing it in my PR.

Reply:
Possible this does not matter -- Pass through is coming, we should look at refactors like this as "how critical" -- if RHODS doesn't have an issue, I think it is okay to let this one go and just bring it to the client. Maybe there is better filters we can use when we are there.